core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+probe {
+
+  # intellij is for org.virtuslab.ideprobe.config.IntellijConfig - default values below
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    # path = "" // use this config with proper path string if you want to use non-default IntellijConfig
+  }
+
+  # workspace is for Option[org.virtuslab.ideprobe.config.WorkspaceConfig] - default None - comments added as a reference
+  # workspace {
+  #   path { // for org.virtuslab.ideprobe.dependencies.Resource
+  #     uri {
+  #       str = ""
+  #     }
+  #     path = "" // for java.nio.file.Path interface - path can be passed as a simple string
+  #     cause {
+  #       s = ""
+  #     }
+  #   }
+  #   existing = "" // for java.nio.file.Path interface - path can be passed as a simple string
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  # }
+
+  # resolvers id for org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers - default values below
+  resolvers {
+    intellij {
+      repositories = []
+    }
+    plugins { }
+    jbr {
+      repositories = []
+    }
+    retries = 0
+  }
+
+  # driver is for org.virtuslab.ideprobe.config.DriverConfig - default values below
+  driver {
+    launch {
+      command = []
+      timeout = "30.seconds"
+    }
+    check {
+      errors {
+        enabled = false
+        includeMessages = [".*"]
```

Comment:
In fact parts of the docs that currently resides in Markdown files could be moved here to the comments, where it'd be more expected (from my PoV as Scala library user)

Reply:
Can be done on a subsequent PR tho

Reply:
Sure, if you have a specific idea in mind, please create an issue for that. Just to create a separate PR for changes in docs / code comments.

Reply:
#254